### PR TITLE
Integrate copr builder

### PR DESCRIPTION
**Pull request by johnsonw**
Integrate copr builder

Signed-off-by: Will Johnson <wjohnson@whamcloud.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,7 +21,7 @@ jobs:
         - include/travis/copr-deploy.sh prepare
         - ./travis_wait "./include/travis/run_in_centos7_docker.sh include/travis/copr-deploy.sh build_srpm"
```
[the remark this answers is not in the document]
Comment (johnsonw) [reply .travis.yml]: We don't need the `cd` step for online-help right?

[the remark this answers is not in the document]
Comment (jgrund) [reply .travis.yml]: Why not?

[the remark this answers is not in the document]
Comment (jgrund) [reply .travis.yml]: Hrrm, it is running currently: https://travis-ci.org/whamcloud/Online-Help/builds/517515252

[the remark this answers is not in the document]
Comment (johnsonw) [reply .travis.yml]: I'm used to seeing it at the bottom of the file and just missed it. I'll move it to the bottom.

**Pull request merged**